test/support/ast_data.ex

```diff
@@ -154,7 +154,20 @@ defmodule Gradient.AstData do
     {expression, _} =
       :erl_parse.mapfold_anno(
         fn anno, acc ->
-          {{:erl_anno.line(anno) - acc, :erl_anno.column(anno)}, acc}
+          line =
+            case :erl_anno.line(anno) - acc do
+              line when is_integer(line) and line >= 0 ->
+                line
+            end
```

Comment:
as a `:erl_anno.line()` is a non-negative integer, gradient  complains when we do this subtraction without ensuring result is non-neg